tests/unit/BusinessTable.spec.ts

```diff
@@ -167,3 +168,79 @@ describe('Business Table', () => {
     })
   }
 })
+
+// *** FUTURE: get this working
+// ATM, local rules are mocked (eg, wrapper.vm.notAffiliated()), but not the actual rules in the mixin.
+// It's probably this: https://vitest.dev/guide/mocking.html#mocking-pitfalls.
+// Maybe try vi.mock() to mock the imported mixin?
+describe.skip('Business Table - rule evaluation', () => {
```

Comment:
I spend quite some time trying to get this working. The idea was to mock (override) the rules, and then clear them one by one, to verify that the map/reduce function was working correctly.

However, while I could mock the rule in `wrapper.vm` (the component), this was not overriding the rules in the mixin :(

Anyway, someone in the future can look at this if they have time and feel like it.

Reply:
Wonder if we should add this as a checkbox somewhere so that we don't forget about it Sev. Anyway, OK with me 👍

Reply:
Nah. There are lots of other unit tests we could write. Mostly I didn't want to throw away this code which partly works :)